src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Button, Form } from 'react-bootstrap';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type State = {
+  id: number;
+  todoTitle: string;
+  userId: number;
+  isUserChoosed: boolean;
+  isTodoInclude: boolean;
+};
+
+type Props = {
+  users: User[];
+  setNewTodo: (todo: Todo) => void;
+  length: number;
+};
+
+export class TodoForm extends React.Component<Props, State> {
+  state: State = {
+    id: this.props.length + 1,
```

Comment:
if some todos are deleted, the length will be decreased and id will be repeated. It will be better to collect all id's and find the max id, then add 1 to that value.